### PR TITLE
add policy relay option in UI

### PR DESCRIPTION
**Pull request by Retropex**

### CODE REVIEW

**Comment by kn0wmad** [comment]
Is there a reason you did not also add `datacarriersize`?

Also, have you tested this?

File: scripts/services/getConfig.ts [hunk 1]
```diff
@@ -124,6 +124,18 @@ export const getConfig: T.ExpectedExports.getConfig = async (effects) => {
...
+      type: "boolean",
+      name: "Datacarrier",
+      description: "Relay and mine data carrier transactions",
+      default: true,
+    },
+    permitbaremultisig: {
+      type: "boolean",
+      name: "Permitbaremultisig",
+      description: "Relay non-P2SH multisig",
+      default: false,
```
Comment (remcoros) [inline scripts/services/getConfig.ts:137]: The default in Bitcoin Core is still 'true', not sure if it's ok to change this from the default

Comment (MattDHill) [reply scripts/services/getConfig.ts:137]: No, given that this is Bitcoin Core, we think it appropriate to ship the same defaults as Core.

**Comment by Retropex** [comment]
@kn0wmad No none, that's why I just added it :)

Yes, I was able to test and everything is functional, do I have to make a test report or something like that?

File: scripts/services/getConfig.ts (hunk 1 above)
Comment (Retropex) [reply scripts/services/getConfig.ts:148]: I will change it if this is a problem, but know that P2MS is no longer used for payment purposes but almost exclusively to add arbitrary data to the chain in the worst possible way.

Also be aware that this relay policy applies only to new output created and not to the already existing one.

The developers in charge of Bitcoin seem to want to change the default value but unfortunately shitcoiners have [sabotaged the PR](https://github.com/bitcoin/bitcoin/pull/28217) to be able to continue to spam the chain.

Comment (remcoros) [reply scripts/services/getConfig.ts:148]: I understand the reasoning and on my personal node I'm definitely tweaking some of these settings! But as a developer and provider of the software you have to stay neutral unless absolutely necessary, and in that case it should be communicated and documented clearly why it's steering away from the defaults. 

It's very good to provide options to the user, but it's the user themself who should decide if he wants to change options different than the consensus reached in the Core repository.

Comment (Retropex) [reply scripts/services/getConfig.ts]: It's good, the default value has been reset to true

**Comment by kn0wmad** [comment]
> @kn0wmad No none, that's why I just added it :)
> 
> Yes, I was able to test and everything is functional, do I have to make a test report or something like that?

No, we just like to know that something has been tested before we take a look at it.  We are still discussing our strategy in regard to these config options/alternate node installs at a higher level.  We appreciate your patience while we figure out the best way forward.

**Review by MattDHill — APPROVED** [review]

**Comment by MattDHill** [comment]
I just realized these options are not in the correct place in config. These are mempool options and should be listed under the mempool section

**Comment by MattDHill** [comment]
Closing in favor of https://github.com/Start9Labs/bitcoind-startos/pull/140 which addresses the comment above and also cleans some things up. Will be sure to give you credit @Retropex, just didn't want to delay getting these features shipped.

**Pull request closed without merging**